packages/noports_core/lib/src/common/types.dart

```diff
@@ -1,12 +1,21 @@
 import 'package:noports_core/sshrv.dart';
 
-typedef SshrvGenerator<T> = Sshrv<T> Function(String, int, {int localSshdPort});
+typedef SshrvGenerator<T> = Sshrv<T> Function(
+  String,
+  int, {
+  required int localPort,
+  required bool bindLocalPort,
```

Comment:
Client-side we now run an rv which binds to a local port on one side, and talks to the rvd on the other; the ssh tunnel is then created by sshing to the local port that the local sshrv is listening on